src/details/ArborX_DetailsDistributor.hpp

```diff
@@ -299,13 +305,13 @@ class Distributor
 
       dest_buffer_mirror =
           ArborX::Details::create_layout_right_mirror_view_and_copy(
-              typename ImportView::memory_space(), dest_buffer);
+              space, typename ImportView::memory_space(), dest_buffer);
```

Comment:
Should not initialize, check.

Reply:
`create_layout_right_mirror_view_and_copy` doesn't initialize anyway.